### PR TITLE
wetland and riparian page edits

### PR DESCRIPTION
**Pull request by rkelson**
change download links
add 'These datasets have a varied update schedule but are updated continuously.' for updates
add or change related links
I plan to get a deploy ready and have DNR review

### CODE REVIEW

**Comment by steveoh** [comment]
You're going to want to use the packagedata include instead of the hub stuff you are trying. https://github.com/agrc/gis.utah.gov/blob/master/_includes/packagedata.html. 

```
{% include packagedata.html name="VotingPrecincts" info=page.VotingPrecincts %}
```

The download url and label will create the link for you. https://github.com/agrc/gis.utah.gov/blob/master/_includes/packagedata.html#L10

**Comment by rkelson** [comment]
I'm guessing a page with 3 related datasets can't have a combination of hub and package data included? In other words 2 external downloads and one opendata download?

**Comment by steveoh** [comment]
from https://app.netlify.com/sites/gis-utah-gov/deploys/6407ddec8370e20008ba6493

image

**Comment by steveoh** [comment]
The us fish and wildlife page has direct download links that would be more seamless.

https://www.fws.gov/wetlands/Data/State-Downloads/UT_shapefile_wetlands.zip
https://www.fws.gov/wetlands/Data/State-Downloads/UT_geodatabase_wetlands.zip

File: data/water/wetlands/index.html [hunk 1]
```diff
@@ -11,7 +11,7 @@
 Wetlands:
   downloads:
     - url: "https://www.fws.gov/node/264847"
-    label: "U.S. Fish & Wildlife Service Downloads"
+      label: "U.S. Fish & Wildlife Service Downloads"
```
Comment (steveoh) [inline data/water/wetlands/index.html:14]: this was a syntax error not a netlify build problem

File: data/water/wetlands/index.html [hunk 2]
```diff
@@ -56,65 +36,35 @@
     <h3 id="Wetlands">Wetlands</h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
-    <img class="inline-text-left outline" src="{% link images/wetlands2.png %}" alt="Wetlands" loading="lazy" />
-    <p>Utah Wetlands is derived from the U.S. Fish and Wildlife Service’s <a
-        href="https://www.fws.gov/wetlands/">National Wetlands Inventory</a> (NWI). NWI attributes wetlands with codes
-      from the Cowardin Classification System; codes can be interpreted <a
-        href="https://www.fws.gov/wetlands/data/wetland-codes.html">online</a>. Cowardin classes in this derived layer
+    <p>Utah Wetlands is derived from the U.S. Fish and Wildlife Service’s
```
[the remark this answers is not in the document]
Comment (rkelson) [reply data/water/wetlands/index.html:39]: Yup but I will likely replace the text with what they suggest. Sending the draft page to them today

**Comment by rkelson** [comment]
this should be good to go

**Review by steveoh — APPROVED** [review]

**Pull request merged**